www/src/components/shell/ConfigureMyCloudButton.js

```diff
@@ -17,7 +21,7 @@ export default function ConfigureMyCloudButton() {
     },
   })
 
-  if (!data) return
+  if (!data || !isOnboarded) return
```

Comment:
@nickcernera This will make button appear only if user has `ONBOARDED` state (this is the case after user skips demo or marks it as complete). Is this what you mean in ENG-907?